Tasks/DeleteFiles/DeleteFiles.ts

```diff
@@ -0,0 +1,91 @@
+/// <reference path="../../definitions/vso-task-lib.d.ts" />
+
+import path = require('path');
+import os = require('os');
+var tl = require('vso-task-lib');
+
+// contents is a multiline input containing glob patterns
+var contents: string[] = tl.getDelimitedInput('Contents', '\n');
```

Comment:
base on the code, looks like the "Contents" can be folder or file? even the task is named "DeleteFiles".